src/clients/python/sams_client/schemas/destinations.py

```diff
@@ -0,0 +1,17 @@
+"""
```

Comment:
The `docstring` needs to be after the module variable, as that's where sphinx expects them here.
Can  you also add a `schemas` section to the client docs and generate the docs to make sure they are generated properly:
`cd docs && make livehtml`

Reply:
Added schemas section in client docs